ddtrace/opentelemetry/tracer.go

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package opentelemetry
+
+import (
+	"context"
+
+	oteltrace "go.opentelemetry.io/otel/trace"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+var _ oteltrace.Tracer = (*oteltracer)(nil)
+
+type oteltracer struct {
+	name     string
+	cfg      oteltrace.TracerConfig
+	provider *TracerProvider
+	ddtrace.Tracer
+}
+
+//todo:can we use config.SpanKind in some way
+func (t *oteltracer) Start(ctx context.Context, spanName string, opts ...oteltrace.SpanStartOption) (context.Context, oteltrace.Span) {
+	var ssConfig = oteltrace.NewSpanStartConfig(opts...)
+	var optsLocal []ddtrace.StartSpanOption
+	if !ssConfig.NewRoot() {
```

Comment:
I'm interpreting `NewRoot` to mean that this span being created should be the root span, not that a separate root span should be created with this as the child. This current implementation will create 2 spans, right?

Reply:
No, if `!ssConfig.NewRoot()`== false, we will create a span with `ChildOf(ctx)` option. In such case newly created span will propagated values from the context. If it is true, the created span is created as a root